docs/user/config-file/v2.rst

```diff
@@ -456,6 +456,43 @@ See :doc:`/build-customization` for more details.
 :Default: ``{}``
 
 
+build.commands
+``````````````
+
+Specify a list of commands that Read the Docs will run on the build process.
+When ``build.commands`` is used, none of the :term:`pre-defined build jobs` will be executed.
+(see :doc:`/build-customization` for more details).
+This allows you to run custom commands and control the build process completely.
+The ``output/`` directory (relative to the checkout's path) will be uploaded and hosted by Read the Docs.
```

Comment:
I do wonder if we want to use something like `output/html/`, so that we have a more obvious path towards supporting other types? We could always just `output_pdf` in the future, but that will be weird that it isn't `output_html`..

Reply:
I had this same feedback. It would be great to have an `output/{html,pdf,epub}/` from the start.

Reply:
Yeah, sounds good to me!

Reply:
I went ahead and used `_readthedocs/html`